src/index.js

```diff
@@ -1,73 +1,3 @@
-import Raw from './raw';
-import Client from './client';
+const Knex = require('./knex');
 
```

Comment:
Rationale behind this change: it it very counter-intuitive to write tests for a file named "index.js" which is in general supposed to be a catalogue and do not contain logic on its own.